packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -533,9 +544,10 @@ function TableRow({item, children, ...otherProps}) {
           {
             'is-active': isPressed,
```

Comment:
The TableView "is-active" styles seem to persist even when the row is no longer pressed.
Specific reproduction:
1. Using option+command+Enter, select a bunch of rows in the "selectionBehavior: replace" story
2. Press up/down arrow which should deselect all rows except the one you are now focusing 
3. Note that the previously selected rows are now grey

![image](https://user-images.githubusercontent.com/9661127/135675874-37011638-e941-40c3-9eff-a86d5b156dbf.png)
Happens for ListView as well